### DISCUSSION TITLE
之前买的专业版，现在变成免费版了怎么弄

### DISCUSSION THREAD

**Discussion by xiaolin6456** [getlantern/lantern-forum-cn#342] [Q&A]
之前买的专业版，现在变成免费版了怎么弄

**Comment by MaxMKMax** [comment]
@xiaolin6456 您好, 请您试试看重新登入能吗? 你是哪个系统? IOS? Android? Window? Mac?
如果不能的话请您提供我邮件和购买单据. 我来帮您我帮你问付款部门. 看看他们是否可以手动升级您.

**Reply by hnliupeng** [reply]
你好，我的账号“hnliupeng@gmail.com”也是，因为我之前几天重复删除安装，之前的专业版现在也变成免费版了，而且我也忘记了之前购买的激活码了，附件是我的购买订单截图，2023年10月26号买的，续费了一年的
![123](https://github.com/getlantern/lantern-forum-cn/assets/39369465/dd447bb2-0d6b-4050-b69a-4b926de57f11)

**Reply by MaxMKMax** [reply]
@xiaolin6456 好的. 我帮您提交上去了.

**Reply by MaxMKMax** [reply]
@hnliupeng 

客服说您可以尝试正常的授权方法（即电子邮件代码链接或设备代码链接）

**Reply by hnliupeng** [reply]
已经可以了，谢谢

**Comment by ZHANGJIAMING19911991** [comment]
我也是一样，前几天买的专业版，解决了闪退，现在变免费版，邮箱也登录不了。

**Reply by MaxMKMax** [reply]
@ZHANGJIAMING19911991 请您提供我邮件和购买单据. 我来找客服帮发您激活码.

**Reply by ZHANGJIAMING19911991** [reply]
你好，刚刚自己无意中解决恢复了，感谢~

**Reply by MaxMKMax** [reply]
@ZHANGJIAMING19911991 嗯嗯. 好的. 不客气. 谢谢您支持蓝灯!

**Comment by Yuemingdu** [comment]
到底有没在修？什么时候能恢复？都没准信的怎么搞的？

**Reply by MaxMKMax** [reply]
@Yuemingdu 
您好. 很多用户都恢复正常了. 
你帮我提交各应用内报告吧. 给我票号. 让他们来看还有什么问题. 谢谢.

**Comment by hnliupeng** [comment]
谢谢，通过邮箱重新申请授权的途径，现在已经恢复了
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年4月3日 3:33
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: hnliupeng ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 之前买的专业版，现在变成免费版了怎么弄 (Discussion #342)


@hnliupeng<https://github.com/hnliupeng>

客服说您可以尝试正常的授权方法（即电子邮件代码链接或设备代码链接）

―

**Reply by MaxMKMax** [reply]
@hnliupeng 嗯嗯. 好的. 不客气. 谢谢您支持蓝灯.
